### PR TITLE
Feature/curl support

### PR DESCRIPTION
**Pull request by restonslacker**
address issues in #29.  principally removing the dependence on a system installation of the curl binary by using the cran package curl

### CODE REVIEW

**Comment by eddelbuettel** [comment]
The Travis break is "mine" as I prefer the old-school explicit Travis.  Please add

```
  - ./travis-tool.sh install_aptget r-cran-curl
```

I will update the Travis file once we're done.

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ before_install:
 
 install:
   - ./travis-tool.sh install_r RJSONIO
+  - ./travis-tool.sh install_r curl
```
Comment (eddelbuettel) [inline .travis.yml]: If both go to `install_r` you can just have multiple args on one line.

But it is faster to do `install_aptget r-cran-curl` which should (!!) be resolvable.  Else I address.

**Comment by restonslacker** [comment]
`git commit --amend -m "change to install_aptget r-cran-curl per suggestion from @eddelbuettel"`
=P

**Comment by eddelbuettel** [comment]
last commit looks good

need to get an updated digest package out the door, this may be next.  getting to the point where I'd need a unified GH issue dashboard :-/

**Comment by restonslacker** [comment]
thanks!

there is one, but maybe it's a little too basic for you: https://github.com/issues

**Comment by eddelbuettel** [comment]
yes, I should probably use that more.

**Comment by eddelbuettel** [comment]
I made one post-PR commit to update a few things, now `R CMD check` is again happy.

You may need to pull this, and if we want to keep the other PR alive it may need some aligning.

**Comment by restonslacker** [comment]
WRT the changeLog, support for "address" pushes was removed as well.

**Pull request merged**